src/rprblender/engine/context.py

```diff
@@ -127,6 +131,9 @@ def clear_scene(self):
 
         self.images = {}
 
+        self.motion_cache = {}
```

Comment:
if this no longer has deformation, it should be named transform_cache or matrix_cache